codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -53,6 +53,11 @@
 
         /**
          * Signal an error and release any retained buffers.
+         *<p>
+         * This method might be called multiple times.
```

Comment:
I think we should find a way to guarantee the contract that only a single call is made to EITHER error() or complete().  WDYT?  I'm assuming the problem is failure of the write promise?